### PR TITLE
run all -e -E -L commands given, in order

### LINKED ISSUE #23665: CLI: all `--eval <code>` should evaluate
Currently we only evaluate the last `--eval <code>` argument. This makes it impossible to build up a sequence of code to run via separate `--eval` arguments, which is often useful. Our current behavior is also at odds with Perl and Ruby which are generally considered to be good models for useful CLI behavior:
```sh
$ perl -e 'print("this\n");' -e 'print("that\n");'
this
that
$ ruby -e 'print("this\n");' -e 'print("that\n");'
this
that
```
In Python, the first occurrence of `-c` terminates option parsing, which we do not do, so following Python here would not make sense. A more conservative choice would be erroring on multiple `--eval` options, which would allow us to change the behavior in the future without breaking code.

### PR DESCRIPTION
**Pull request by vtjnash**
fix #23665

### CODE REVIEW

**Review by StefanKarpinski — COMMENTED** [review]
Nice. The change to calling display seems good but I feel like one will generally want to have the trailing newline. In case one doesn't, it's still always possible to use `-e` and call `display` / `show` in the expression.

**Comment by vtjnash** [comment]
Yeah, there's just a slight problem that there's no way to `display` a newline, it would need to be done by the `Display`.

**Comment by StefanKarpinski** [comment]
What's wrong with just calling `println`?

**Comment by vtjnash** [comment]
It doesn't get written to the Display.

**Comment by StefanKarpinski** [comment]
Why does that matter?

**Comment by StefanKarpinski** [comment]
There was talk of a pass-through display wrapper type, which would allow `display(MIMEData{"text/plain"}("\n"))` to express what would be wanted here.

File: base/options.jl [hunk 1]
```diff
@@ -6,9 +6,7 @@ struct JLOptions
     banner::Int8
     julia_home::Ptr{UInt8}
     julia_bin::Ptr{UInt8}
-    eval::Ptr{UInt8}
-    print::Ptr{UInt8}
-    load::Ptr{UInt8}
+    commands::Ptr{Ptr{UInt8}} # (e)eval, (E)print, (L)load
```
Comment (iblis17) [inline base/options.jl]: In C struct, this field named as `cmds`, it's inconsist.
Is this intended?

**Pull request merged**